src/UI/FileTreeView.re

```diff
@@ -75,31 +72,55 @@ let nodeView = (~font: Core.Types.UiFont.t, ~fg, ~bg, ~node: FsTreeNode.t, ()) =
       <FontIcon
         color=fg
         icon={isOpen ? FontAwesome.folderOpen : FontAwesome.folder}
-        backgroundColor=bg
+        backgroundColor=Colors.transparentWhite
       />
     | _ => <icon />
     };
 
   <View style=Styles.item>
     <icon />
-    <Text text={node.displayName} style={Styles.text(~fg, ~bg, ~font)} />
+    <Text text={node.displayName} style={Styles.text(~fg, ~font)} />
   </View>;
 };
 
 module TreeView = TreeView.Make(FsTreeNode.Model);
 
-let make = (~title, ~tree: FsTreeNode.t, ~onNodeClick, ~state: State.t, ()) => {
+let make =
+    (
+      ~title,
+      ~tree: FsTreeNode.t,
+      ~focus: option(string),
+      ~onNodeClick,
+      ~state: State.t,
+      (),
+    ) => {
   [@warning "-27"]
   let State.{theme, uiFont as font, _} = state;
 
   let bg = state.theme.sideBarBackground;
   let fg = state.theme.sideBarForeground;
 
+  let focus =
+    switch (focus, state.workspace) {
+    | (Some(path), Some(workspace)) =>
+      let re =
+        Str.regexp_string(workspace.workingDirectory ++ Filename.dir_sep);
+      let localPath = path |> Str.replace_first(re, "");
+
+      switch (FsTreeNode.findNodesByLocalPath(localPath, tree)) {
+      | `Success(nodes) => Some(nodes)
+      | `Partial(_)
+      | `Failed => None
+      };
+
+    | _ => None
+    };
```

Comment:
Note to self: Extract to a function with a descriptive name (`fsPathToNodePath`?)